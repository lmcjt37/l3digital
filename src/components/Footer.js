import React from 'react'

const Footer = () => (
    <footer id="footer">
        <div className="inner">
            <ul className="icons">
                <li>
                    <a
                        href="https://twitter.com/l3digital_uk"
                        className="icon alt fa-twitter"
                    >
                        <span className="label">Twitter</span>
                    </a>
                </li>
                <li>
                    <a
                        href="https://github.com/l3digitaluk"
                        className="icon alt fa-github"
                    >
                        <span className="label">GitHub</span>
                    </a>
                </li>
                <li>
                    <a
                        href="https://www.linkedin.com/in/l3-digital-8a369b180/"
                        className="icon alt fa-linkedin"
                    >
                        <span className="label">LinkedIn</span>
                    </a>
                </li>
            </ul>
            <ul className="copyright">
                <li>&copy; {new Date().getFullYear()} L3 Digital</li>
                <li>
                    Design: <a href="https://html5up.net">HTML5 UP</a>
                </li>
            </ul>
        </div>
    </footer>
)

export default Footer
