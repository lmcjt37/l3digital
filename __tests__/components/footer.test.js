import Footer from '../../src/components/Footer'

describe('Footer tests', () => {
    test('check it renders correctly with snapshot', () => {
        const tree = render(<Footer />)
        expect(tree).toMatchSnapshot()
    })
})
