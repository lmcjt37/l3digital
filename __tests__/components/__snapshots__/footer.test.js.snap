// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests check it renders correctly with snapshot 1`] = `
<footer
  id="footer"
>
  <div
    className="inner"
  >
    <ul
      className="icons"
    >
      <li>
        <a
          className="icon alt fa-twitter"
          href="https://twitter.com/l3digital_uk"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="label"
          >
            Twitter
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon alt fa-github"
          href="https://github.com/l3digitaluk"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="label"
          >
            GitHub
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon alt fa-linkedin"
          href="https://www.linkedin.com/in/l3-digital-uk/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="label"
          >
            LinkedIn
          </span>
        </a>
      </li>
    </ul>
    <ul
      className="copyright"
    >
      <li>
        Â© 
        2019
         L3 Digital
      </li>
      <li>
        Design: 
        <a
          href="https://html5up.net"
        >
          HTML5 UP
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
